configs/katello/4.7.yaml

```
@@ -0,0 +1,110 @@
+---
+:project: katello
+:github_org: katello
+:releases:
+  :4.7:
+    :redmine_version_id: 1613
+:prior_releases:
+  :4.6:
+    :redmine_version_id: 1561
+:code_name: Whiskey Sour
+:mash_scripts:
+  - katello-4.7-el8
+  - katello-candlepin-4.7-el8
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-4.7
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :foreman-packaging:
+    :branch: rpm/3.5
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :hammer-cli-katello:
+    :branch: 1.7.z
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :katello-host-tools:
+    :branch: 4.2.2
+    :repo: https://github.com/Katello/katello-host-tools.git
+  :katello-certs-tools:
+    :branch: 2.9.0
+    :repo: https://github.com/Katello/katello-certs-tools.git
+  :foreman_virt_who_configure:
+    :branch: 0.5.10
+    :repo: https://github.com/theforeman/foreman_virt_who_configure.git
+  :smart_proxy_container_gateway:
+    :branch: 1.0.6
+    :repo: https://github.com/Katello/smart_proxy_container_gateway.git
+  :katello-selinux:
+    :branch: 4.0.2
+    :repo: https://github.com/Katello/katello-selinux.git
+:ignores:
+:strict_keys: true
+:gpg_key: F120E13E
```

I just created the key. Please update this
```suggestion
:gpg_key: 8f5ca95b
```